doc/vector-extra/riscv-crypto-vector-extra-zvkgs.adoc

```diff
@@ -0,0 +1,32 @@
+[[zvkgs,Zvkgs]]
+=== `Zvkgs` - Vector-Scalar GCM/GMAC
+
+`Zvkgs` depends on `Zvkg`, it extends the existing `vghsh.vv` and `vgmul.vv` instructions with new vector-scalar variants: `vghsh.vs` and `vgmul.vs`.
+
+Instructions to enable the efficient implementation of parallel versions of GHASH~H~ which is used in Galois/Counter Mode (GCM) and
```

Comment:
This is not a complete sentence.

More importantly, it's not clear to me how these `vs` instructions for GHASH are helpful, because parallelized GHASH requires multiplying by powers of the key (like H, H^2, H^3, H^4, ...).  Does that not preclude the use of these `vs` instructions?

Reply:
There are multiple different implementations of parallel GHASH. 
One implementation on 4 block in paralell uses a vector of 4 constants H^4 in the loop body which could leverage the `.vs`. It still needs a reduction with H, H^2, H^3 in the loop epilog.

Reply:
I rephrased things a bit (in the upcoming v0.0.4). The introduction sentence is re-used from https://github.com/riscv/riscv-crypto/blob/main/doc/vector/riscv-crypto-vector-zvkg.adoc.

I need to work on an example of use of these instructions.